Sources/GRPCHTTP2Core/Compression/Zlib.swift

```diff
@@ -37,29 +37,24 @@ enum Zlib {
 extension Zlib {
   /// Creates a new compressor for the given compression format.
   ///
-  /// This compressor is only suitable for compressing whole messages at a time. Callers
-  /// must ``initialize()`` the compressor before using it.
-  struct Compressor {
+  /// This compressor is only suitable for compressing whole messages at a time.
+  final class Compressor {
```

Comment:
Could we use a `~Copyable` type here instead?

Reply:
I don't think that'd solve the problem as the types can still be moved around.